src/NavBar.tsx

```diff
@@ -0,0 +1,47 @@
+import './navBar.css'
+import React from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
```

Comment:
These are unused imports. Use withStyles instead. Then you can declare the css in the same file like
```
const styles = {
    toolBar: {
        display: flex,
        color: white,
    },
}

class NavBar extends React.Component {
    ...
}

export default withStyles(styles)(NavBar)